plugin/metrics/vars/report.go

```diff
@@ -20,7 +20,7 @@ func Report(req request.Request, zone, rcode string, size int, start time.Time)
 	typ := req.QType()
 
 	RequestCount.WithLabelValues(zone, net, fam).Inc()
-	RequestDuration.WithLabelValues(zone).Observe(float64(time.Since(start) / time.Millisecond))
+	RequestDuration.WithLabelValues(zone).Observe(time.Since(start).Seconds())
```

Comment:
ah that is nice. Didn't know about that. (in this context). Prolly convert it back to milliseconds?